tests/test_cli.py

```diff
@@ -427,3 +427,83 @@ def test_clean_no_task(tmp_empty):
 
     assert result.exit_code == 2
     assert "Error: Invalid value for 'FILENAME'" in result.output
+
+
+output_test = """\
+# ## first
+
+cf = 1
+
+# ## second
+
+import fastparquet
+from pathlib import Path
+import pandas as pd
+
+j = open('file.txt', 'w')
+j.close()
+
+df = pd.DataFrame()
+df.to_csv('file_csv.csv')
+df.to_parquet('my.parquet')
+Path('write_text').write_text("stf")
+Path('write_byte').write_bytes('stf')
+
+
+"""
+
+
+def test_refactor_product_warning_for_output_files(tmp_empty):
+    Path('nb.py').write_text(output_test)
+    args = 'nb.py'
+
+    runner = CliRunner()
+    result = runner.invoke(cli.refactor, args)
+
+    assert result.exit_code == 0
+    assert 'open' in result.output
+    assert 'to_csv' in result.output
+    assert 'to_parquet' in result.output
+    assert 'write_text' in result.output
+    assert 'write_bytes' in result.output
+
+
+output_in_comment_test = """\
+# ## first
+
+cf = 1
+
+# ## second
```

Comment:
You can also change this test to be a single cell, and have some read command in it to make sure we're not having false positives.

Reply:
Yeah here we might have to use regrex since open() can be read only and not outputting stuff.